.github/workflows/test-suite.yml

```diff
@@ -34,8 +34,16 @@ jobs:
     secrets: inherit
     uses: ./.github/workflows/cleanup-test-env.yml
 
+  acc-tests-latest-published:
```

Comment:
I am struggling to see the value of adding this job to run daily, given that the results will be the same unless there is a new patch release from TF. 
If you can provide more context in the PR or reference where this requirement was discussed that would be great. I am thinking why we cannot rely on our master branch to see if the latest patch version of TF is breaking some functionality in our provider.

Reply:
the use case here is that Hashicorp release a new version, e.g. 1.6.7 and breaks our latest published version, e.g. 1.14.0.    it's true that results won't change while Hashicorp doesn't publish a new version. But the alternative is that we have to be pending (or automate?) when a new version is published and then run on demand.

Reply:
@Zuhairahmed what do you think?
alternatively we can do that, don't run every night but on demand when we know that a new Hashicorp version came out.

Reply:
@AgustinBettati at the moment I'll leave only the option to run it manually